lib/ansible/plugins/callback/logentries.py

```diff
@@ -60,9 +60,13 @@
 import random
 import time
 import codecs
-import ConfigParser
 import uuid
 try:
+    import ConfigParser
+except ImportError:
+    import configparser as ConfigParser
+
```

Comment:
Is this all we have to do for this?  I think in py3, SafeConfigParser has taken over the name that ConfigParser used to inhabit.